internal/stores/metadata.go

```diff
@@ -43,6 +43,7 @@ type (
 		ContractCommon
 		RenewedTo fileContractID `gorm:"index;size:32"`
 
+		HostID uint      `gorm:"index"`
```

Comment:
Any particular reason for this? It makes me quite nervous to have an ID here without a relation. It might end up pointing to an ID that was previously deleted or maybe deleted and recreated.

Reply:
I added it because it's useful for writing manual queries.  I don't mind removing it but I don't really see why we can't add it either because `dbContract` has it. I don't really see why an archived contract isn't more or less an exact copy of the active contract.